api/openstorage.yaml

```diff
@@ -803,16 +815,26 @@ paths:
                      }
                     }
               ```
-          required: true
-          schema:
-            $ref: '#/definitions/VolumeCreateRequest'
-      tags:
-        - Volumes
-      responses:
-        '200':
-          description: UUID of newly created volume
-          schema:
-            $ref: '#/definitions/VolumeCreateResponse'
+            required: true
+            schema:
+              $ref: '#/definitions/VolumeCreateRequest'
+        tags:
+          - Volumes
+        responses:
+          '200':
+            description: UUID of neewly created volume
+            schema:
+              $ref: '#/definitions/VolumeCreateResponse'
+            examples:
+             application/json: |-
+              {
+                 "cmd": "create",
+                 "status": "Ok",
```

Comment:
Do all our APIs return status? What is the 'status' when there is an error? E.g. when you create volumes of the same name. How does the end user check if the command was successful - is it by comparing status == "Ok"?